### PR TITLE
Fix teamcity logging

### PR DESCRIPTION
**Pull request by asika32764**
https://github.com/sebastianbergmann/phpunit/issues/5354

This is a very simple workarounds to fix PhpStorm unit test output.

Before

screenshot 2024-12-02 下午1 32 52

```
##teamcity[testSuiteStarted name='Windwalker\Cache\Test\Storage\MemcachedStorageTest' locationHint='php_qn:///private/var/www/windwalker/framework/packages/cache/test/Storage/MemcachedStorageTest.php::\Windwalker\Cache\Test\Storage\MemcachedStorageTest' flowId='5419']

```

After

screenshot 2024-12-02 下午4 56 00

```
##teamcity[testSuiteStarted name='Windwalker\Cache\Test\Storage\MemcachedStorageTest' locationHint='php_qn:///private/var/www/windwalker/framework/packages/cache/test/Storage/MemcachedStorageTest.php::\Windwalker\Cache\Test\Storage\MemcachedStorageTest' flowId='5419']
##teamcity[testIgnored name='Windwalker\Cache\Test\Storage\MemcachedStorageTest' message='Memcached not supported' duration='0' flowId='5419']
##teamcity[testSuiteFinished name='Windwalker\Cache\Test\Storage\MemcachedStorageTest' message='Memcached not supported' duration='0' flowId='5419']

```

Currently I'm not totally understand the emitter system of phpunit, so I'm not able to add the test ignored message for every methods, only fix the class level skipped actions.

### CODE REVIEW

**Comment by sebastianbergmann** [comment]
As the proposed changes fix a bug, this pull request should target the oldest affected/supported branch, which is `10.5`.

May I kindly ask you to 1) squash your changes into a single commits and 2) target the `10.5` branch? Thank you!

File: src/Logging/TeamCity/TeamCityLogger.php [hunk 1]
```diff
@@ -7,6 +7,7 @@
  * For the full copyright and license information, please view the LICENSE
  * file that was distributed with this source code.
  */
+
```
Comment (sebastianbergmann) [inline src/Logging/TeamCity/TeamCityLogger.php:10]: This looks wrong and unrelated.

File: src/Logging/TeamCity/TeamCityLogger.php [hunk 2]
```diff
@@ -33,23 +35,28 @@
...
 use PHPUnit\Framework\Exception as FrameworkException;
 use PHPUnit\TextUI\Output\Printer;
 
 /**
- * @no-named-arguments Parameter names are not covered by the backward compatibility promise for PHPUnit
+ * @no-named-arguments Parameter names are not covered by the backward compatibility promise for
+ *                     PHPUnit
  *
- * @internal This class is not covered by the backward compatibility promise for PHPUnit
+ * @internal           This class is not covered by the backward compatibility promise for PHPUnit
```
Comment (sebastianbergmann) [inline src/Logging/TeamCity/TeamCityLogger.php:49]: This looks wrong and unrelated.

**Comment by sebastianbergmann** [comment]
I am sorry, but I cannot review this in its current state. Please do not make any unrelated (whitespace) changes and make sure to run `./tools/php-cs-fixer fix` before commit/push.

**Comment by asika32764** [comment]
As the 10.5 is far away from 11.4, I re-open a new PR for it.

**Comment by asika32764** [comment]
https://github.com/sebastianbergmann/phpunit/pull/6066

**Pull request closed without merging**